cypress/e2e/pages/governance/create/index.spec.ts

```diff
@@ -1,6 +1,7 @@
 const viewPorts = ["macbook-16", "ipad-2", "iphone-x"];
 
 viewPorts.forEach((viewPort) => {
+  // todo: rewrite completely to be compatible with cypress 12: tests to not share the context
```

Comment:
Does this share contexts? It's mostly just visual checks

Reply:
yes, it does, 
checks for ocg step 2 relies on step 1, 
checks for ocg step 3 relies both on step 1 and step 2,
negative checks also relies on the previous context